pkg/mount/mount.go

```diff
@@ -111,6 +111,24 @@ func (m *mounter) GetDevicePath(ctx context.Context, volumeID string) (string, e
 }
 
 func (m *mounter) getDevicePathBySerialID(volumeID string) (string, error) {
+	// First try XenServer device paths
+	xenDevicePath, err := m.getDevicePathForXenServer(volumeID)
+	if err != nil {
+		fmt.Printf("Failed to get XenServer device path: %v\n", err)
```

Comment:
These print statements will get printed in the stdout for each volume mount. For an end user, this might create a lot of noise for me in the logs. Assuming that the caller logs the error, maybe we can just remove line or log it at trace level.

Reply:
Also, it would be better to use `fmt.Println` and then `\n` won't be required at the end of the string.